index.bs

```diff
@@ -2230,7 +2230,12 @@ To <dfn>attempt to deliver a debug report</dfn> given an [=attribution report=]
     [=generate an attribution report URL/isDebugReport=] set to true.
 1. Let |data| be the result of executing [=serialize an attribution report=] on |report|.
 1. If |data| is an error, return.
-1. Let |request| be the result of executing [=create a report request=] on |url| and |data|.
+1. Let |newHeaders| be a new [=header list=].
+1. If |report| is an [=aggregatable report=]
+    1. If |report|'s [=aggregatable report/serialized private state token=] is not null,
+        [=header list/append=] a new [=header=] named "`Sec-Attribution-Reporting-Private-State-Token`" to |newHeaders|
+        whose value is |report|'s [=aggregatable report/serialized private state token=].
+1. Let |request| be the result of executing [=create a report request=] on |url|, |data|, and |newHeaders|.
```

Comment:
Seems like this is partly duplicated from the non-debug report version. Can we extract a common algorithm and use it in both places?